staging/src/k8s.io/apiextensions-apiserver/test/integration/basic_test.go

```diff
@@ -332,7 +334,8 @@ func TestInvalidCRUD(t *testing.T) {
 	for _, v := range noxuDefinition.Spec.Versions {
 		noxuResourceClients[v.Name] = newNamespacedCustomResourceVersionedClient("", dynamicClient, noxuDefinition, v.Name)
 
-		noxuWatch, err := noxuResourceClients[v.Name].Watch(metav1.ListOptions{})
+		noxuWatch := noxuResourceClients[v.Name].Watch(metav1.ListOptions{})
```

Comment:
what does noxu stand for?

Reply:
A made-up CRD name, defined staging/src/k8s.io/apiextensions-apiserver/test/integration/fixtures/resources.go.

I have to say, this author is not very creative in creating test data.  If the CRD is given a name "flyingpants" or "sillymarmots", you will not have such a confusion. :-)